examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<ul><li>Truncated <s>regresison</s> <strong>regression</strong> model </li><li>quite simple to <s>impliment</s> <strong>implement</strong></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>fixed</p>

Reply:
<p>There is still a typo in the title: "Truncated <s>regresison</s> <strong>regression</strong> model"</p>

Reply:
<p>should be fixed now. Still learning... I think running the pre-commit checks locally interfered with an unsaved local copy. I'll get there in the end ;)</p>